Experience.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="header">
+        <a id = "myName" href="index.html">David Gonzalez</a>
+    </header>
+    <main class="main">
+        <div>
+            <h1>My name is david and this is a list of my experience.</h1>
+            <ul>
+                <li>
+                    item1
+                    <p>I worked hard at item 1</p>
+                </li>
+                <li>
+                    item2
+                    <p>i worked hard at item 2</p>
+                </li>
+                <li>
+                    item3
+                    <p>i worked hard at item 3</p>
+                </li>
+                <li>
+                    item4
+                    <p>i worked hard at item 4</p>
+                </li>
+            </ul>
+        </div>
+        <nav class = "navigation">
```

Comment:
very cool side nav implementation! as a stretch goal, think about responsiveness of your page and how the structure might adjust as the browser gets smaller and smaller.